backend/src/emissions/schema.py

```diff
@@ -832,7 +834,12 @@ def mutate(root, info, input=None):
             input.fuel_type,
             input.group_share,
         )
-        co2e_cap = co2e / user.working_group.n_employees
+        # error boundary if user has no working group.
+        print(vars(user))
+        if user.working_group is not None:
```

Comment:
I think this case should never occurr, since this endpoint can only be called by a user who is a representative of a group (due to the decorator `@representative_required`) and therefore this user must have a working group assigned. Unless the working group of the representative is not set correctly when they are made the representative which would be a bug

Reply:
I agree, this only occured because of already existing users for which the working group was not set correctly. I reverted the change in [b7ea328](https://github.com/pledge4future/WePledge/pull/236/commits/b7ea328b2bc5f7bcf6038b1590d2e30d6563d31c)